src/search/view/SearchView.ts

```diff
@@ -232,6 +243,27 @@ export class SearchView extends BaseTopLevelView implements TopLevelView<SearchV
 		this.viewSlider = new ViewSlider([this.folderColumn, this.resultListColumn, this.resultDetailsColumn])
 	}
 
+	private async routeToType(type: string): Promise<void> {
+		let latestRestriction = null
+		// contacts do not have restrictions at this time
+		if (type === "mail") {
```

Comment:
if `type` is a more narrow type you could  `switch` on all 3 cases here to make sure we don't forget to add more in the future